UnitTests/GTMSessionFetcherFetchingTest.m

```diff
@@ -1720,6 +1720,40 @@ - (void)testQuickBeginStopFetching {
 #endif
 }
 
+- (void)testQuickBeginDontStopFetching {
```

Comment:
This seems unrelated to the PR at hand; can we simply handle the completion callback-with-error behavior, and assert that cancellation triggers an appropriate error in the fetcher?

Consider having your test access the local HTTP `_testServer` with a "sleep=" query parameter so you can have time to cancel the fetch on the client side.

Reply:
This test directly contrasts with the test before it to verify that the callbacks do get called, where they didn't in the previous test. Isn't that useful?

Reply:
The test before this is testing a specific workaround; testing that the callback is triggered on cancel requires different behavior, which I described in my original comment.